projects/go-lib/src/lib/components/go-icon/go-icon.component.ts

```diff
@@ -1,26 +1,31 @@
-import { Component, Input, OnInit } from '@angular/core';
+import { Component, Input, OnInit, OnChanges } from '@angular/core';
 
 @Component({
   selector: 'go-icon',
   templateUrl: 'go-icon.component.html',
   styleUrls: ['go-icon.component.scss']
 })
-export class GoIconComponent {
+export class GoIconComponent implements OnChanges {
+  classObject: object = {};
+
   @Input() icon: string;
   @Input() iconModifier: string;
   @Input() iconClass: string;
+  @Input() disabled: boolean = false;
```

Comment:
Does it matter that disabled is an html attribute here? Should we change this to be iconDisabled or something instead?

Reply:
I think so long as it doesn't end up being `disabled="false"` we should be okay with it because it follows the html spec. I will make sure that isn't the case.

Reply:
I can confirm this. So long as `[disabled]` is an expression binding I think we should prefer this.